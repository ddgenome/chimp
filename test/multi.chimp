# simple catalytic mechanism with a multisite species test
mechanism "multi.mech"
parameter "multi.par"
task "multi.task"
