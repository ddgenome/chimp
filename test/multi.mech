# simple catalytic mechanism with multisite species
A + 3 @ <- k_Arrhenius(A_Ades, E_Ades) -> k(A_Aads) @@@A;
@@@A -> k_Arrhenius(A_srf, E_srf) B + 3 @;
