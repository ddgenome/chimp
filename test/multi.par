# parameters for multi task
A_Aads	1.0e-1
A_Ades	1.0e12
E_Ades	6.2e4
A_srf	1.0e12
E_srf	6.9e4
