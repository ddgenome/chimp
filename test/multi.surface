x = 0
@@@A    @       @       @@@A    @@@A    @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @       @       @       @@@A    @       @       @       @       @       @       @       @       @       @       @       @       @       
@@@A    @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @       @       @@@A    @@@A    @@@A    @       @       @       @       @@@A    @@@A    @       @       @       @       @       @       @@@A    
@       @       @@@A    @       @       @       @       @       @@@A    @       @@@A    @@@A    @       @       @       @       @       @       @       @       @       @       @@@A    @       @       @@@A    @@@A    @       @       @@@A    
@       @       @@@A    @       @       @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @       @@@A    @@@A    @@@A    @@@A    @       @@@A    @@@A    @       @       @       @@@A    @@@A    @@@A    @@@A    @       @       @@@A    
@       @       @@@A    @       @       @       @       @       @@@A    @       @       @@@A    @@@A    @@@A    @       @       @@@A    @       @@@A    @       @@@A    @@@A    @       @@@A    @       @@@A    @@@A    @       @@@A    @       
@       @       @       @       @       @       @       @       @@@A    @       @       @@@A    @@@A    @@@A    @       @       @       @       @       @       @@@A    @       @       @@@A    @@@A    @       @@@A    @       @@@A    @@@A    
@       @       @@@A    @@@A    @@@A    @@@A    @       @@@A    @       @       @       @       @       @       @       @       @       @       @@@A    @@@A    @@@A    @@@A    @       @@@A    @       @       @@@A    @       @       @       
@       @       @@@A    @       @@@A    @       @       @@@A    @@@A    @       @       @@@A    @@@A    @@@A    @       @       @       @       @       @       @       @@@A    @       @       @       @       @@@A    @@@A    @       @       
@       @@@A    @       @       @       @       @       @       @       @       @@@A    @       @       @       @       @       @       @       @       @       @@@A    @@@A    @       @       @       @       @@@A    @@@A    @       @       
@@@A    @@@A    @       @       @       @@@A    @@@A    @       @       @@@A    @@@A    @@@A    @       @       @       @@@A    @@@A    @       @@@A    @       @@@A    @       @       @       @@@A    @       @       @@@A    @@@A    @       
@       @       @       @       @@@A    @@@A    @       @       @       @       @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @       @       @       @@@A    @@@A    @       @@@A    @       @       
@@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @       @       @       @       @       @       @@@A    @@@A    @@@A    @@@A    @       @       @       @       @       @@@A    @       @       @@@A    @@@A    @       @       
@@@A    @       @       @       @@@A    @       @@@A    @@@A    @       @       @@@A    @       @       @       @@@A    @       @@@A    @@@A    @@@A    @       @       @@@A    @       @@@A    @       @       @       @@@A    @@@A    @@@A    
@@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @@@A    @       @       @@@A    @       @       @@@A    @@@A    @@@A    @       @@@A    @       @@@A    @@@A    @       @       @       @       @@@A    
@       @@@A    @@@A    @       @       @       @@@A    @       @       @       @       @       @       @       @@@A    @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @@@A    @       @@@A    @       @       @@@A    
@       @       @       @       @       @       @       @       @       @@@A    @@@A    @@@A    @@@A    @       @@@A    @       @       @@@A    @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    
@@@A    @       @       @       @       @       @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @@@A    @@@A    @@@A    @       @@@A    @       @       @@@A    @       @@@A    @       @       @       @       
@@@A    @       @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @@@A    @@@A    @@@A    @       @       @       @       @       @       @@@A    @       @@@A    @       @       @       @@@A    
@@@A    @       @       @@@A    @@@A    @@@A    @       @@@A    @       @@@A    @       @       @@@A    @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @       @       @       @       @       @       
@@@A    @       @       @       @       @       @       @       @       @       @       @       @       @       @       @       @       @       @       @@@A    @       @       @       @       @       @       @@@A    @@@A    @       @@@A    
@@@A    @       @@@A    @@@A    @       @       @@@A    @       @       @@@A    @       @       @@@A    @@@A    @       @       @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @       @       @@@A    
@@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @@@A    
@@@A    @       @@@A    @@@A    @@@A    @@@A    @       @       @       @       @       @       @@@A    @       @       @       @       @       @@@A    @       @       @       @@@A    @@@A    @@@A    @       @       @       @       @@@A    
@@@A    @       @       @       @       @@@A    @       @       @@@A    @@@A    @       @@@A    @@@A    @       @@@A    @       @       @       @       @       @       @       @       @       @@@A    @@@A    @@@A    @@@A    @       @@@A    
@       @       @@@A    @       @@@A    @@@A    @       @@@A    @@@A    @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @@@A    @@@A    @       @@@A    @       @@@A    @       @       
@@@A    @@@A    @@@A    @@@A    @       @       @@@A    @@@A    @@@A    @@@A    @       @       @       @@@A    @@@A    @       @@@A    @       @@@A    @@@A    @@@A    @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    
@       @       @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @@@A    @@@A    @       @       @       @@@A    @@@A    @       @       @       @       @       @@@A    @@@A    @@@A    @       @@@A    @@@A    @       @       @       
@       @@@A    @       @       @@@A    @@@A    @@@A    @       @@@A    @       @@@A    @@@A    @@@A    @@@A    @       @@@A    @@@A    @       @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       
@@@A    @@@A    @       @       @@@A    @@@A    @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @       @       @       @       @       @       @@@A    @       @       @       @       @       @@@A    
@@@A    @@@A    @       @@@A    @       @@@A    @@@A    @       @       @@@A    @       @@@A    @@@A    @@@A    @       @       @@@A    @       @       @@@A    @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    

x = 0.00100265
@@@A    @@@A    @       @@@A    @@@A    @       @@@A    @@@A    @       @       @@@A    @@@A    @       @@@A    @@@A    @       @       @       @       @       @       @       @@@A    @@@A    @       @       @       @@@A    @       @       
@@@A    @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @       @       @       @@@A    @       @@@A    @@@A    @       @       @       @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    
@@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @       @       @@@A    @@@A    @       @@@A    @@@A    @       @       @@@A    @@@A    @       @       @       @@@A    @@@A    @       @       @       @@@A    @@@A    @@@A    
@@@A    @       @@@A    @       @       @@@A    @@@A    @@@A    @       @@@A    @       @       @       @       @       @       @       @@@A    @@@A    @       @       @       @@@A    @@@A    @       @       @       @@@A    @       @@@A    
@       @       @@@A    @       @@@A    @@@A    @@@A    @       @       @@@A    @       @       @@@A    @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @       @       @@@A    @@@A    @       
@       @@@A    @@@A    @       @       @@@A    @       @       @       @@@A    @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @@@A    @       @       @@@A    @@@A    @@@A    
@@@A    @@@A    @       @       @@@A    @@@A    @       @       @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @@@A    @@@A    @       @@@A    @       @       @@@A    @       @       @@@A    @@@A    @@@A    
@@@A    @@@A    @       @       @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @       @       @       @@@A    @       
@       @@@A    @       @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @       
@@@A    @@@A    @       @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @@@A    @@@A    @       @       @       @       @       @       @@@A    @@@A    @@@A    @       @       @@@A    
@       @       @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @       @@@A    @@@A    @       @       @       @@@A    
@       @       @       @@@A    @@@A    @@@A    @       @@@A    @@@A    @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @@@A    @@@A    @@@A    @       @       @       @       @       @       @@@A    
@@@A    @       @       @       @       @@@A    @       @@@A    @       @       @@@A    @       @@@A    @@@A    @@@A    @       @       @       @@@A    @       @       @       @@@A    @@@A    @@@A    @       @       @@@A    @@@A    @@@A    
@       @       @@@A    @       @       @       @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @       @       @       @       @@@A    @@@A    @       @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       
@       @@@A    @@@A    @       @@@A    @       @       @       @       @@@A    @       @       @@@A    @       @       @       @       @@@A    @@@A    @       @       @       @       @       @       @       @@@A    @       @       @       
@@@A    @@@A    @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @       @@@A    @       @       @@@A    @       @       @       @       @@@A    @@@A    @@@A    @       @@@A    @@@A    @       @       
@       @@@A    @@@A    @@@A    @       @       @       @       @@@A    @@@A    @       @       @       @@@A    @@@A    @       @       @       @       @       @       @       @       @@@A    @@@A    @@@A    @       @       @       @       
@       @       @@@A    @       @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @       @       @@@A    @       @       @       @       @       @       @@@A    @@@A    @       @       @@@A    @       @       
@@@A    @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @@@A    @       @       @@@A    @@@A    @       @@@A    @       @       @       @@@A    @@@A    @       
@@@A    @       @@@A    @@@A    @@@A    @       @@@A    @       @       @       @       @       @       @       @@@A    @@@A    @       @@@A    @@@A    @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    
@@@A    @       @@@A    @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @       @       @@@A    @@@A    @@@A    @@@A    @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    
@@@A    @       @@@A    @       @       @@@A    @@@A    @       @@@A    @       @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @       @       @       @       @       @       @       @@@A    @@@A    @@@A    
@       @       @@@A    @       @       @       @@@A    @@@A    @@@A    @       @       @       @       @       @       @@@A    @@@A    @@@A    @       @       @@@A    @       @       @       @       @       @       @       @       @@@A    
@       @@@A    @@@A    @@@A    @       @       @       @       @@@A    @@@A    @@@A    @       @       @       @@@A    @       @       @       @       @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @       @       @       @       
@       @@@A    @@@A    @       @@@A    @@@A    @       @       @@@A    @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @       @@@A    @@@A    @@@A    
@@@A    @@@A    @       @@@A    @@@A    @       @       @       @@@A    @@@A    @       @       @       @       @       @@@A    @@@A    @       @       @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    
@       @       @@@A    @@@A    @@@A    @       @@@A    @@@A    @       @@@A    @       @       @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @@@A    @       @@@A    @@@A    @       @@@A    @@@A    
@       @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @       @       @       @@@A    @@@A    @@@A    @       @       @@@A    @@@A    @       @       @       @       @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @       
@       @@@A    @@@A    @       @@@A    @       @       @@@A    @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @@@A    @       @       @       @       @       @       @       
@@@A    @@@A    @       @@@A    @       @       @@@A    @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @@@A    @@@A    @       @       

x = 0.0100014
@@@A    @@@A    @@@A    @@@A    @       @       @       @       @@@A    @       @@@A    @@@A    @       @       @       @@@A    @       @@@A    @       @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    
@       @       @@@A    @@@A    @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @       @@@A    @@@A    @       @@@A    @       @@@A    @       @       @       @       @@@A    @       @       @       @       @@@A    @@@A    @       
@       @       @@@A    @       @       @@@A    @       @       @       @@@A    @@@A    @       @       @@@A    @@@A    @@@A    @       @@@A    @       @       @       @       @@@A    @@@A    @@@A    @       @       @       @@@A    @       
@       @       @       @       @@@A    @@@A    @       @       @@@A    @       @@@A    @       @@@A    @       @       @       @       @       @       @       @@@A    @       @@@A    @       @@@A    @       @       @       @@@A    @       
@@@A    @@@A    @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @@@A    @       @       @       @@@A    @       @       @       @@@A    @@@A    @       @       @       @@@A    @       @@@A    @@@A    @@@A    
@@@A    @       @       @       @@@A    @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @@@A    @@@A    @       @       @@@A    @       @       @       @@@A    @@@A    @       @       @@@A    
@@@A    @@@A    @@@A    @@@A    @       @       @@@A    @@@A    @       @       @@@A    @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @@@A    @       @       @       @       @@@A    @@@A    @@@A    
@@@A    @       @       @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @       @       @       @@@A    @       @       @@@A    @       @       @@@A    @@@A    @       @@@A    @@@A    @@@A    
@       @       @       @       @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @       @@@A    @@@A    @@@A    @       @@@A    @       @       @       @@@A    @@@A    @       @       @@@A    @       @@@A    @@@A    @       @       
@       @       @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @@@A    @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @@@A    @       @@@A    @@@A    @       
@       @@@A    @@@A    @       @@@A    @       @       @@@A    @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @       @@@A    @       @       @       @       @       @@@A    @@@A    @@@A    @       @@@A    @       
@@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @       @       @       @       @       @       @       @       @@@A    @       @@@A    @@@A    @       @@@A    @       
@       @@@A    @@@A    @@@A    @       @       @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @       @       @       @       @@@A    @@@A    @       @       @@@A    @@@A    @       @@@A    
@@@A    @       @@@A    @@@A    @@@A    @       @       @       @@@A    @@@A    @@@A    @       @       @@@A    @@@A    @@@A    @@@A    @       @       @       @@@A    @       @@@A    @@@A    @       @       @@@A    @       @       @@@A    
@       @       @       @@@A    @@@A    @       @       @       @@@A    @       @       @       @       @@@A    @@@A    @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @       @@@A    @@@A    @@@A    
@@@A    @@@A    @       @@@A    @@@A    @       @@@A    @       @@@A    @       @       @       @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @       @       @       @       @       @@@A    @@@A    @@@A    
@@@A    @       @@@A    @@@A    @@@A    @       @@@A    @       @@@A    @       @@@A    @@@A    @@@A    @       @       @       @@@A    @@@A    @       @       @       @@@A    @       @       @       @       @       @       @       @       
@       @       @@@A    @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @@@A    @       @@@A    @@@A    @       @       @       @       @       @       @       @@@A    @       
@       @       @@@A    @       @@@A    @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @@@A    @       @@@A    @       @       @       @       @       @@@A    @       @       @@@A    @       
@       @@@A    @@@A    @@@A    @       @       @       @       @       @       @       @       @@@A    @@@A    @       @@@A    @       @@@A    @@@A    @@@A    @@@A    @       @       @       @@@A    @@@A    @       @       @@@A    @       
@@@A    @@@A    @       @@@A    @       @       @@@A    @@@A    @       @       @       @       @@@A    @       @@@A    @@@A    @       @@@A    @       @       @       @@@A    @       @@@A    @@@A    @@@A    @@@A    @       @@@A    @@@A    
@       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @       @       @@@A    
@       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @       @@@A    @@@A    @       @       @       @@@A    @       @       @       
@       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @       @       @@@A    @@@A    @@@A    @       @@@A    @       @@@A    @@@A    @       @       @       @@@A    @@@A    @@@A    @@@A    @       @@@A    @       
@       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @       @@@A    @       @       @       @@@A    @       @       @       @       @@@A    @@@A    @       @       @       @@@A    @       
@@@A    @@@A    @       @@@A    @@@A    @@@A    @       @@@A    @       @@@A    @@@A    @@@A    @       @       @       @       @       @       @@@A    @       @       @@@A    @       @       @@@A    @       @       @       @@@A    @@@A    
@@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @       @@@A    @@@A    @       @       @       @       @@@A    @       @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @@@A    @       
@       @@@A    @       @       @       @       @       @       @       @       @       @@@A    @@@A    @@@A    @       @@@A    @@@A    @       @@@A    @@@A    @       @       @@@A    @@@A    @@@A    @@@A    @       @@@A    @@@A    @       
@       @       @@@A    @@@A    @@@A    @       @       @       @@@A    @       @@@A    @       @@@A    @       @       @       @       @       @@@A    @@@A    @@@A    @       @@@A    @       @       @@@A    @@@A    @@@A    @       @       
@       @       @       @@@A    @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @@@A    @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @       @       @@@A    @       @@@A    @@@A    @@@A    

x = 0.0200011
@       @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @       @       @@@A    @       @@@A    @@@A    @@@A    @       @       @@@A    @       @@@A    @@@A    @@@A    @       @       @       @       @@@A    @       @       
@       @@@A    @       @@@A    @@@A    @       @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @@@A    @@@A    @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @       
@       @       @       @       @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @       @@@A    @       @       @@@A    @       @@@A    @@@A    @@@A    @       @       @       @       @@@A    @       @@@A    @@@A    @@@A    
@@@A    @       @       @       @@@A    @@@A    @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @       @       @@@A    @       @       @@@A    @@@A    
@       @       @       @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @@@A    @       @       @       @@@A    @@@A    @       @@@A    @       @       @       @       
@@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @@@A    @@@A    @@@A    @       @@@A    @       @       
@@@A    @       @@@A    @       @       @@@A    @       @@@A    @@@A    @       @       @       @@@A    @       @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       
@@@A    @@@A    @@@A    @       @       @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @@@A    @       @       @       @@@A    @@@A    @       @@@A    @       @@@A    @       @       @@@A    @       
@       @       @       @       @       @@@A    @       @@@A    @@@A    @       @@@A    @       @@@A    @       @       @       @       @       @       @@@A    @       @@@A    @@@A    @@@A    @       @@@A    @       @       @@@A    @       
@       @       @@@A    @@@A    @       @       @@@A    @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @@@A    @       
@@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @@@A    @@@A    @@@A    @       @       @       @       @       @       @       @@@A    @       @@@A    @       @@@A    @@@A    @       @@@A    @       @       
@       @       @       @@@A    @@@A    @       @       @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @       @@@A    @       @       @       @       @@@A    @@@A    @       @       @@@A    @       @@@A    @       @       
@@@A    @@@A    @       @@@A    @@@A    @       @       @       @       @@@A    @@@A    @@@A    @@@A    @       @@@A    @       @@@A    @@@A    @@@A    @       @@@A    @@@A    @       @       @@@A    @@@A    @       @@@A    @@@A    @       
@       @@@A    @       @       @@@A    @       @       @       @@@A    @       @       @       @       @       @       @       @       @@@A    @@@A    @@@A    @       @       @       @       @       @@@A    @@@A    @@@A    @@@A    @       
@       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @@@A    @@@A    @       @       @       @       @@@A    @@@A    @@@A    @@@A    @       @       @       @       @@@A    @@@A    @       @       
@@@A    @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @       @@@A    @       @       @@@A    @       @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @@@A    @       @@@A    @       @       @       
@@@A    @       @       @       @       @       @@@A    @@@A    @       @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @@@A    @@@A    @@@A    @       @@@A    @@@A    @@@A    
@@@A    @       @       @       @       @       @       @@@A    @       @@@A    @       @@@A    @       @@@A    @       @       @       @       @@@A    @@@A    @       @       @       @@@A    @       @       @       @@@A    @@@A    @@@A    
@       @       @       @       @       @       @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @       @@@A    @       @       @       @       @       @@@A    @@@A    @@@A    @       @@@A    @@@A    
@       @       @       @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @       @       @       @@@A    @       @@@A    @       @       @       @@@A    @@@A    @@@A    @       @       @       @       @@@A    @       @@@A    @@@A    
@       @       @@@A    @       @@@A    @@@A    @       @       @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @       @       @       @       @       @       @@@A    @@@A    @       @       @@@A    @@@A    @@@A    @@@A    
@@@A    @       @@@A    @@@A    @@@A    @@@A    @       @       @       @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @       @       @       @@@A    @       @       @       @       @       @       @@@A    
@@@A    @       @@@A    @       @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @       @       @       @       @       @@@A    @@@A    @@@A    @       @       @@@A    
@       @@@A    @@@A    @@@A    @@@A    @       @       @       @       @@@A    @@@A    @       @       @@@A    @@@A    @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @       @@@A    @@@A    
@       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @@@A    @@@A    @       @       @       @@@A    @@@A    @@@A    @       @       @       @       @       @       @       @@@A    @@@A    
@@@A    @@@A    @       @@@A    @       @       @       @@@A    @       @       @       @       @@@A    @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @       @@@A    @@@A    @@@A    @       
@@@A    @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @@@A    @@@A    @       @       @@@A    @       @       @       @@@A    @       @@@A    @       @       @       @       @@@A    @@@A    @@@A    @@@A    @       
@@@A    @@@A    @@@A    @       @@@A    @       @       @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @@@A    @@@A    @@@A    @       @@@A    @@@A    @       @       @       @       @       @       
@@@A    @@@A    @       @       @@@A    @@@A    @       @@@A    @       @       @       @       @@@A    @@@A    @       @       @       @       @       @@@A    @@@A    @@@A    @@@A    @@@A    @       @       @       @@@A    @       @@@A    
@       @@@A    @@@A    @       @@@A    @       @@@A    @@@A    @       @@@A    @@@A    @@@A    @       @@@A    @       @       @       @@@A    @@@A    @       @       @@@A    @@@A    @       @       @       @       @@@A    @@@A    @@@A    

