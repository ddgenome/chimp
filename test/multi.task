# -*- text -*-
# multi input
begin model multi
  output "multi.out"
  begin integrator kmc
    size 30
    surface_file "multi.surface"
    begin state
      begin quantity
	p[A] = 1.0e5		# Pa
	@[@@@A] = 1.5e-1
      end quantity
      begin output
	1.0e-3 1.0e-2 2.0e-2
      end output
      begin reactor batch
	temperature 4.0e2	# K
	pressure 1.0e5		# Pa
	volume 3.0e-5		# m^3
	sites 2.0e19
	rate_numerator molecules
	rate_denominator sites
	fluid_quantity pressure
      end reactor
    end state
  end integrator
end model
