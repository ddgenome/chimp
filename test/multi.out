# multi
# x	@	@@@A	A	B	steps
0.000000e+00	5.500000e-01	1.500000e-01	1.000000e+05	0.000000e+00	0
1.002653e-03	4.866667e-01	1.711111e-01	9.931683e+04	6.054437e+02	315
1.000142e-02	4.866667e-01	1.711111e-01	9.383511e+04	6.087163e+03	2997
2.000110e-02	4.900000e-01	1.700000e-01	8.790340e+04	1.202297e+04	5904
